src/persist-client/src/read.rs

```diff
@@ -695,13 +695,6 @@ where
         Ok(Listen::new(self, as_of).await)
     }
 
-    /// Returns a [`BatchFetcher`], which does not hold since or seqno
-    /// capabilities.
-    #[instrument(level = "debug", skip_all, fields(shard = %self.machine.shard_id()))]
-    pub async fn batch_fetcher(self) -> BatchFetcher<K, V, T, D> {
```

Comment:
Ah, yeah, I think this one was added when shard_source lived outside src/persist-client. Now that it's merged in, def makes sense to me to get rid of this